src/main/resources/liquibase/workbench_changelog/19_3_0.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v19.3.0-1">
+		<preConditions onFail="MARK_RAN">
+			  <sqlCheck expectedResult="1">
+					SELECT count(1) > 0 FROM (
+										SELECT 1
+										FROM role_permission rp
+										INNER JOIN permission p ON rp.permission_id = p.permission_id
+										WHERE p.name IN ('GERMPLASM', 'MANAGE_GERMPLASM')
+										GROUP BY rp.role_id
+									) AS count;
+			  </sqlCheck>
+        </preConditions>
+		<comment>
+			Removing Merge germplasm permission for roles that do not have Full permissions.
```

Comment:
@cuenyad please review it again because some conditions are using inventory permissions

Reply:
@clarysabel the MG_MANAGE_INVENTORY permission is for create lots in Manage Germplasm module.